contents/blog/posthog-changelog.md

```diff
@@ -19,6 +19,35 @@ Interested about what new features, fixes and updates have just launched on Post
 
 > Some new features you read about below may still be in beta, behind a feature flag, or only available to paying users. Want to see which betas are currently available? [Check the roadmap](/roadmap)!
 
+## March 3, 2023
+
+### Hedgehog Mode
+![hog mode](../images/blog/array/hog_walk.gif)
+
+Let's get the biggest and most important updates out of the way first. We've updated our hedgehog mode, which you can access via the help menu in the app, to include some extra features. 
+
+You can now control Max ([that's his name](/handbook/company/team)) using the WASD keys, and we've also added a physics engine to give him a proper sense of gravity and inertia. You can make him jump with `W`, or spin with `S`. 
+
+### React library
+Our Head of Product, Luke, decided to stretch his engineering skills this week and added a new React library which makes it easy to get data about traffic and product usage from React apps, into PostHog.
+
+Even better, Luke also started testing his writing skills by updating [the React docs](/docs/sdks/react) to include everything you need to get started with the full PostHog suite, including feature flags!
```

Comment:
```suggestion
You can now integrate PostHog into a React app using the `PostHogProvider`. After that PostHog provides a bunch of hooks include `usePostHog` to access the initialized client, and `useFeatureEnabled` and `useFeaturePayload` for using feature flags. Checkout [the React docs](/docs/sdks/react) or [the Next.js docs][docs/integrate/third-party/next-js] for more information.
```

Reply:
How about this for an image?

![ray-so-export](https://user-images.githubusercontent.com/22766134/222223659-7999152d-2429-44b4-86a8-dab9407c2d78.png)

Reply:
Welcome to tweak the image here: https://ray.so/#code=ZnVuY3Rpb24gQXBwKCkgewogICAgcmV0dXJuICgKICAgICAgICA8UG9zdEhvZ1Byb3ZpZGVyIGFwaUtleT17cHJvY2Vzcy5lbnYuUE9TVEhPR19BUElfS0VZfT4KICAgICAgICAgICAgPFJvb3QgLz4KICAgICAgICA8L1Bvc3RIb2dQcm92aWRlcj4KICApCn0&language=jsx&theme=candy&title=PostHog+React&padding=32